spec/controlling-tasks.md

```diff
@@ -0,0 +1,117 @@
+Controlling Tasks {#sec-controlling-tasks}
+=====================
+
+Tasks scheduled through the {{Scheduler}} interface can be controlled with a
+{{TaskController}} by passing the associated {{TaskSignal}}
+{{AbortController/signal}} as an {{SchedulerPostTaskOptions/signal|option}} to
+{{Scheduler/postTask()}}. The {{TaskController}} interface supports aborting
+and changing the priority of a task or group of tasks.
+
+Note: {{TaskSignal}} inherits from {{AbortSignal}} and can be used in APIs that
+accept an {{AbortSignal}}. Additionally, {{Scheduler/postTask()}} accepts an
+{{AbortSignal}}, which can be useful if dynamic prioritization is not needed.
+
+The `TaskPriorityChangeEvent` Interface {#sec-task-priority-change-event}
+---------------------
+
+<pre class='idl'>
+  [Exposed=(Window, Worker)]
+  interface TaskPriorityChangeEvent : Event {
+    constructor(DOMString type, TaskPriorityChangeEventInit priorityChangeEventInitDict);
+
+    readonly attribute TaskPriority previousPriority;
+  };
+
+  dictionary TaskPriorityChangeEventInit : EventInit {
+    required TaskPriority previousPriority;
+  };
+</pre>
+
+A {{TaskPriorityChangeEvent}} object has a <dfn for=TaskPriorityChangeEvent>previousPriority</dfn>
```

Comment:
Events are unfortunately unlike every other interface, because their constructor is auto-generated. We have this concept of "what the attribute was initialized to" and "initialize the attribute" which goes against our usual rule of using associated values (like your previousPriority `<dfn>`).

 The most modern way to define their getter steps, while still sticking to this "initialized to" infrastructure, is probably like https://wicg.github.io/app-history/#dom-apphistorynavigateevent-canrespond .

Reply:
Yeah a lot of attributes (most of them?) need actual programmatic getter/setter steps that do clever things. Events are the most common case where they really want dumb getters/setters though, so we currently have this strange "initialzied to" convention. Oh well.